### ISSUE TITLE
autodocsumm fails to load with sphinx 2.4

### ISSUE THREAD

**Issue by scottgigante** [Chilipp/autodocsumm#22]
sphinx.ext.autodoc.Signature has been removed in Sphinx 2.4.
```
Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/scprep/envs/dev/lib/python3.7/site-packages/sphinx/registry.py", line 453, in load_extension
    mod = import_module(extname)
  File "/home/docs/checkouts/readthedocs.org/user_builds/scprep/envs/dev/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/docs/checkouts/readthedocs.org/user_builds/scprep/envs/dev/lib/python3.7/site-packages/autodocsumm/__init__.py", line 28, in <module>
    from sphinx.ext.autodoc import Signature, get_documenters
ImportError: cannot import name 'Signature' from 'sphinx.ext.autodoc' (unknown location)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/scprep/envs/dev/lib/python3.7/site-packages/sphinx/cmd/build.py", line 275, in build_main
    args.tags, args.verbosity, args.jobs, args.keep_going)
  File "/home/docs/checkouts/readthedocs.org/user_builds/scprep/envs/dev/lib/python3.7/site-packages/sphinx/application.py", line 245, in __init__
    self.setup_extension(extension)
  File "/home/docs/checkouts/readthedocs.org/user_builds/scprep/envs/dev/lib/python3.7/site-packages/sphinx/application.py", line 397, in setup_extension
    self.registry.load_extension(self, extname)
  File "/home/docs/checkouts/readthedocs.org/user_builds/scprep/envs/dev/lib/python3.7/site-packages/sphinx/registry.py", line 456, in load_extension
    raise ExtensionError(__('Could not import extension %s') % extname, err)
sphinx.errors.ExtensionError: Could not import extension autodocsumm (exception: cannot import name 'Signature' from 'sphinx.ext.autodoc' (unknown location))

Extension error:
Could not import extension autodocsumm (exception: cannot import name 'Signature' from 'sphinx.ext.autodoc' (unknown location))
```

**Comment by Chilipp** [comment]
Thanks @scottgigante! I am working on it and up to now modified the run-requirements of the conda-forge package to `sphinx <2.4.0` (see https://github.com/conda-forge/autodocsumm-feedstock/pull/8)

**Comment by Chilipp** [comment]
Hi @scottgigante! This issue has been fixed in #23 and I just pushed a new release ([0.1.12](https://github.com/Chilipp/autodocsumm/releases/tag/v0.1.12)) to [PyPi](https://pypi.org/project/autodocsumm/0.1.12/) (and conda-forge just got merged in https://github.com/conda-forge/autodocsumm-feedstock/pull/9).

**Comment by scottgigante** [comment]
Thanks so much for the quick fix!
